frontend/src/v5/ui/routes/dashboard/projects/federations/editFederationModal/editFederationModal.component.tsx

```diff
@@ -56,24 +56,44 @@ export const EditFederationModal = ({
 		);
 	}, [containers]);
 
+	const partitionContainersByQuery = (
+		containersToPartition: IContainer[],
+		query: string,
+	) : [IContainer[], IContainer[]] => (
+		partition(
+			containersToPartition,
+			(container) => (
+				Object.values(pick(container, ['name', 'code', 'type']))
+					.some((value) => (value as string).toLocaleLowerCase().includes(query.toLocaleLowerCase()))),
+		)
+	);
```

Comment:
I dont think is a good  idea to re implement the filtering logic.  I see two ways of going about this:
1. have an onFiltered event in the list that returns the filtered containers.
2.  reimplement  partitionContainersByQuery using  filterContainers  from '@/v5/store/containers/containers.helpers';

Reply:
I went with option #2, so I now use the `filterContainers` function inside `partitionContainers`